config.go

```diff
@@ -125,6 +128,27 @@ func ConfigValue(v interface{}, dst interface{}) error {
 			return fmt.Errorf("unhandled type %+v for float64 conversion", reflect.TypeOf(vt))
 		}
 
+	case *color.Color:
+		switch vt := v.(type) {
+		case int64:
```

Comment:
I would simplify this. It's really unlikely someone tries to convert an integer/float/bool to a valid color. Supporting only hex-strings seems reasonable to me.